cwl/src/main/scala/wdl4s/cwl/model.scala

```diff
@@ -207,12 +208,15 @@ case class Dirent(
   * @param envDef
   */
 case class EnvVarRequirement(
-  `class`: Witness.`"EnvVarRequirement"`.T,
-  envDef:
-    Array[EnvironmentDef] :+:
-    Map[EnvironmentDef#EnvName, EnvironmentDef#EnvValue] :+:
-    Map[EnvironmentDef#EnvName, EnvironmentDef] :+:
-    CNil)
+                              `class`: Witness.`"EnvVarRequirement"`.T,
```

Comment:
I'd guess because you're not initializing to a default value here and you are in the other spots